Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift

```diff
@@ -66,12 +81,47 @@ extension TabTrayV2ViewController: UITableViewDataSource {
             viewModel.removeTab(forIndex: indexPath)
         }
     }
+    
+    @objc func didTapToolbarAddTab(_ sender: UIButton) {
+        viewModel.addTab()
+        dismissTabTray()
+    }
+    
+    @objc func didTapToolbarDelete(_ sender: UIButton) {
+        let controller = AlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        controller.addAction(UIAlertAction(title: Strings.AppMenuCloseAllTabsTitleString, style: .default, handler: { _ in self.viewModel.closeTabsForCurrentTray() }), accessibilityIdentifier: "TabTrayController.deleteButton.closeAll")
+        controller.addAction(UIAlertAction(title: NSLocalizedString("Cancel", comment: "Label for Cancel button"), style: .cancel, handler: nil), accessibilityIdentifier: "TabTrayController.deleteButton.cancel")
+        controller.popoverPresentationController?.sourceView = sender
+        controller.popoverPresentationController?.sourceRect = sender.bounds
+        present(controller, animated: true, completion: nil)
+    }
+    
+    func hideDisplayedTabs( completion: @escaping () -> Void) {
+           let cells = tableView.visibleCells
+
+           UIView.animate(withDuration: 0.2,
```

Comment:
Indentation seems really off, is it just github? 🤔

Reply:
This is the indentation copied from the old implementation, so I didn't change it.